projects/packages/forms/src/contact-form/class-editor-view.php

```diff
@@ -61,7 +61,7 @@ public static function grunion_media_button() {
 	public static function mce_external_plugins( $plugin_array ) {
 		$plugin_array['grunion_form'] = Assets::get_file_url_for_environment(
 			'_inc/build/contact-form/js/tinymce-plugin-form-button.min.js',
-			'modules/contact-form/js/tinymce-plugin-form-button.js'
+			'jetpack_vendor/automattic/jetpack-forms/src/contact-form/js/tinymce-plugin-form-button.js'
```

Comment:
Could we load those 2 references relatively from the `dist` dir instead?
https://github.com/Automattic/jetpack-forms/tree/1facd39f1b5d34ca8ed31b7f9d0fde2d8d94c540/dist/contact-form/js